src/screens/Home/InvestmentsSection.tsx

```diff
@@ -118,8 +126,8 @@ const InvestmentsSection: FC<IInvestmentsSection> = () => {
       setStakingLockedEndTime(lockedEndTime);
 
       const enabled = stakingInfo?.contractState === 1 && remoteInfo?.featureStaking;
-      const stakingMaxTotal = BigNumber.from(stakingInfo.maxStakeTotal.toString());
-      const totalStaked = BigNumber.from(stakingInfo.totalStaked.toString());
+      const stakingMaxTotal = BigNumber.from(stakingInfo?.maxStakeTotal.toString());
+      const totalStaked = BigNumber.from(stakingInfo?.totalStaked.toString());
```

Comment:
If stakingInfo isn't available, what does 
```suggestion
      const stakingMaxTotal = BigNumber.from(stakingInfo?.maxStakeTotal.toString());
      const totalStaked = BigNumber.from(stakingInfo?.totalStaked.toString());
```
Become?

Reply:
If stakingInfo isn't available
=> if (!stakingInfo || !remoteInfo) return;

stakingInfo?.maxStakeTotal => In case stakingInfo object available but maxStakeTotal not available.